spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableErrorChannel.java

```
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.binding;
+
+import org.springframework.integration.channel.PublishSubscribeChannel;
+
+/**
+ * A {@link Bindable} component that represents an error channel.
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+public class BindableErrorChannel extends NoopBindable {
```

Should return the error channel via `getOutputs`